steampy/login.py

```diff
@@ -117,11 +115,53 @@ def _assert_valid_credentials(login_response: Response) -> None:
             raise InvalidCredentials(login_response.json()['message'])
 
     def _perform_redirects(self, response_dict: dict) -> None:
-        parameters = response_dict.get('transfer_parameters')
+        parameters = response_dict.get('transfer_info')
         if parameters is None:
             raise Exception('Cannot perform redirects after login, no parameters fetched')
-        for url in response_dict['transfer_urls']:
-            self.session.post(url, parameters)
+        for pass_data in parameters:
+            pass_data['params']['steamID'] = response_dict['steamID']
+            self.session.post(pass_data['url'], pass_data['params'])
 
     def _fetch_home_page(self, session: Session) -> Response:
         return session.post(SteamUrl.COMMUNITY_URL + '/my/home/')
+
+    def _update_steam_guard(self, login_response: Response) -> bool:
```

Comment:
should return None as it just changes internal state, and we never use its value if it is ok

Reply:
Done